gef-ui/server/controllers/application-details/index.js

```diff
@@ -55,15 +55,19 @@ function buildBody(app, previewMode) {
   const exporterUrl = `/gef/application-details/${app.id}`;
   const facilityUrl = `/gef/application-details/${app.id}/facilities`;
 
-  return {
+  const appBody = {
     application: app,
     status: app.status,
     isAutomaticCover: app.submissionType === DEAL_SUBMISSION_TYPE.AIN,
     exporter: {
       status: app.exporterStatus,
-      rows: mapSummaryList(app.exporter, exporterItems(exporterUrl, {
-        showIndustryChangeLink: app.exporter.details.industries && app.exporter.details.industries.length > 1,
-      }), previewMode),
+      rows: mapSummaryList(
+        { details: app.exporter }, // wrap in details because mapSummaryList relies this.
+        exporterItems(exporterUrl, {
+          showIndustryChangeLink: app.exporter?.industries && app.exporter?.industries.length > 1,
```

Comment:
wouldn't this work? `showIndustryChangeLink:  app.exporter?.industries?.length > 1,`